bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/FileCollection.java

```diff
@@ -41,15 +41,18 @@ public FileCollection(Supplier<Path> base, PassageFileExtension extension) {
 		this.extensions = extension;
 	}
 
-	public Collection<Path> get() throws LicensingException {
-		try (Stream<Path> all = filesIn(base.get())) {
+	public Collection<Path> get() throws LicensingException, IllegalArgumentException {
```

Comment:
IllegalArgumentException is RuntimeException

is must not be declared as it is a part of interface

IllegalArgumentException means _development_, not _runtime_ error.